### PR TITLE
Upload build artifacts

### PR DESCRIPTION
**Pull request by buberdds**
Built on top of #1621

### CODE REVIEW

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -33,5 +33,17 @@ jobs:
...
+        run: |
+          echo "SHORT_SHA=$(git rev-parse --short HEAD)" >> "$GITHUB_OUTPUT"
       - name: Build app
         run: yarn build
+      - name: Upload build artifacts
+        # Upload build artifacts on push event.
+        if: github.event_name == 'push'
+        uses: actions/upload-artifact@v3
+        with:
+          name: oasis-wallet-web-${{ steps.vars.outputs.SHORT_SHA }}
```
Comment (tjanez) [inline .github/workflows/ci-build.yml:48]: Does it make sense to omit the `-web` part or will we build different artifacts for different form factors (i.e. web, browser extension and mobile app)?

Comment (buberdds) [reply .github/workflows/ci-build.yml:48]: For now I will remove `-web` suffix. Long term it depends if we want to have extension build artifacts too (not sure about mobile)

Comment (buberdds) [reply .github/workflows/ci-build.yml]: According to @lubej we can have mobile build artifacts too, but an issue might be signing with certificate.

Comment (tjanez) [reply .github/workflows/ci-build.yml]: Ok, in this case, let's add the `-web` suffix.

**Review by tjanez — APPROVED** [review]

**Pull request merged**